transport/grpc/outbound_test.go

```diff
@@ -200,3 +202,82 @@ func TestCallStreamWithInvalidPeer(t *testing.T) {
 		}.Error(),
 	)
 }
+
+func TestCallSvcMatch(t *testing.T) {
+	tests := []struct {
+		msg         string
+		serverPort  string
+		headerKey   string
+		headerValue string
+		wantErr     bool
+	}{
+		{
+			msg:         "test call service match success",
```

Comment:
nit: adding `test` to these message names are redundant, let's simplify!

Reply:
Removed.